client/src/components/cart/Cart.jsx

```diff
@@ -53,6 +53,10 @@ function Cart() {
     );
   }
 
+  useEffect(() => {
```

Comment:
We had agreed that storing the cart in the local storage was causing a weird behaviour, so we decided to remove it, why are you storing the cart in local store again? @ilkckn fyi @JannaChalohlu